golang.go

```diff
@@ -0,0 +1,19 @@
+// pacage docummentation
+package main
+
+import "context"
+
+// here is a line coment
```

Comment:
Typo inside comment in word `coment`. Possible corrections:  `comment` (61), `moment` (33), `cement` (17)